### ISSUE TITLE
Memory leaks

### ISSUE THREAD

**Issue by sanyashvets** [websitenmaker12/Advanced-Genetics#168]
08:52:53] [Server thread/ERROR]: Detected ongoing potential memory leak. 20400 packets have leaked. Top offenders
[08:52:53] [Server thread/ERROR]:    AdvGen_Network : 20303
[08:52:53] [Server thread/ERROR]:    FML : 97

What should I to to fix?

**Comment by MJRLegends** [comment]
Yeah i have the same message too, but idk what the fix or cause is
